matrix/docker/matrix_converter.py

```diff
@@ -122,7 +122,7 @@ def zarr_to_mtx(zarr_root):
     mtx_path = os.path.join(temp_dir, "matrix.mtx")
     dataframe = pandas.DataFrame(zarr_root.expression[:])
 
-    sparse_mat = scipy.sparse.coo_matrix(dataframe.values)
+    sparse_mat = scipy.sparse.coo_matrix(dataframe.values).transpose()
```

Comment:
Rather than just calling .transpose() in both the code and tests, it would be nice if we could formulate some type of assertion in the tests for this code change